build.sbt

```diff
@@ -20,7 +20,7 @@ lazy val wartremoverSettings =
       routes.in(Compile).value ++
         (baseDirectory.value ** "*.sc").get ++
         Seq(sourceManaged.value / "main" / "sbt-buildinfo" / "BuildInfo.scala"),
-    wartremoverErrors in (Test, compile) --= Seq(Wart.NonUnitStatements, Wart.Null, Wart.PublicInference)
+    wartremoverErrors in (Test, compile) --= Seq(Wart.Any, Wart.NonUnitStatements, Wart.Null, Wart.PublicInference)
```

Comment:
I added this in because I was getting a load of `Any` errors when compiling the tests with the new 
```scala
  def sendEvent[A](auditType: String, detail: A, transactionName: String)(
    implicit ec: ExecutionContext,
    hc: HeaderCarrier,
    writes: Writes[A],
    request: Request[_]
  ): Unit
```
in the `AuditService`. I think `Any`'s were creeping in with scalamock due to the type parameter in the function.

Reply:
or possibly the `_` in the `Request`